crates/redstone/src/types/feed_id.rs

```diff
@@ -41,3 +42,26 @@ impl From<Vec<u8>> for FeedId {
         Self(buff)
     }
 }
+
+impl fmt::Display for FeedId {
```

Comment:
there are functions like that in the [network/as_str.rs](https://github.com/redstone-finance/rust-sdk/blob/main/crates/redstone/src/network/as_str.rs)